src/collar/oc_settings.lsl

```diff
@@ -509,7 +509,7 @@ default
                 g_iBootup=FALSE;
             }
             llSetTimerEvent(0);
-        }else llSetTimerEvent(0.25); // send 1 setting per quarter second
+        }else llSetTimerEvent(0.15); // 0.25 send 1 setting per quarter second
```

Comment:
Let's either get this comment in sync with the code or make it more general so we don't have to keep it in sync.

Reply:
Done